.eleventy.js

```diff
@@ -12,13 +12,25 @@ module.exports = function(eleventyConfig) {
     return DateTime.fromJSDate(dateObj).toFormat("dd LLL yyyy");
   });
 
+  // https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-date-string
```

Comment:
For v8.dev, I decided to also use this in the HTML, hence the name. It’s a valid format for `<time datetime=…>`.

```html
Published <time datetime="2018-09-18">18 September 2018</time>
```

Happy to submit a follow-up PR if you think this markup makes sense for post dates.

Reply:
:+1: from me!